src/main/client/src/components/@commons/DataSheet.tsx

```diff
@@ -17,92 +16,118 @@ type DataSheetProps = {
-      </Flex>
-      {organization.description && <Paragraph>{organization.description}</Paragraph>}
-      {organization.interests && (
-        <>
-          <Heading size="md" marginBottom={0}>
-            A kör tevékenységei közé tartozik:
-          </Heading>
-          <Wrap marginTop={2}>
-            {organization.interests.map((interest) => (
-              <Tag colorScheme={organization.color} variant="solid">
-                {interest}
-              </Tag>
-            ))}
-          </Wrap>
-        </>
-      )}
-      <Wrap marginTop={10} justify={['center', 'center', 'flex-start']}>
-        {organization.website && (
-          <LinkButton href={organization.website} external leftIcon={<LinkIcon />} colorScheme={organization.color}>
-            Weboldal
-          </LinkButton>
+export const DataSheet: React.FC<DataSheetProps> = ({ organization }) => (
+  <>
+    {!organization.hideName && <Heading>{organization.name}</Heading>}
+    <Flex flexDir={{ base: 'column-reverse', sm: 'row' }} justify="space-between" align="center" mt={{ base: 2, sm: 5 }}>
+      <VStack alignItems="flex-start" alignSelf={{ base: 'flex-start', sm: 'center' }}>
+        {organization.established && (
+          <DataField icon={<FaBusinessTime />} label="Alapítva">
+            <Box>{organization.established}</Box>
+          </DataField>
         )}
-        {organization.application && (
-          <LinkButton href={organization.application} external leftIcon={<EditIcon />} colorScheme="brand">
-            Jelentkezés
-          </LinkButton>
+        {organization.email && (
+          <DataField icon={<FaAt />} label="E-mail">
+            <Box>{organization.email.replace('@', '[at]')}</Box>
+          </DataField>
         )}
-        {organization.facebook && (
-          <LinkButton href={organization.facebook} external leftIcon={<FaFacebook />} colorScheme="facebook">
-            Facebook
-          </LinkButton>
+        {organization.members && (
+          <DataField icon={<FaUsers />} label="Létszám">
+            <Box>{organization.members} fő</Box>
+          </DataField>
         )}
-        {organization.instagram && (
-          <LinkButton href={organization.instagram} external leftIcon={<FaInstagram />} colorScheme="purple">
-            Instagram
-          </LinkButton>
+        {organization.interests && (
+          <Flex flexWrap="wrap">
+            {organization.interests.map((interest) => (
+              <Box p={0.5}>
+                <Tag colorScheme={organization.color} variant="solid" key={interest}>
+                  {interest}
+                </Tag>
+              </Box>
+            ))}
+          </Flex>
         )}
-      </Wrap>
-    </>
-  )
+      </VStack>
+      {generateLogo(organization)}
+    </Flex>
+    {organization.descriptionParagraphs && generateParagraphs(organization.descriptionParagraphs)}
+
+    <Wrap marginTop={10} justify={{ base: 'center', md: 'flex-end' }}>
```

Comment:
Replaced the `justify={['center', 'center', 'flex-start']}` here to `justify={{ base: 'center', md: 'flex-end'`. Reason? Using objects with named breakpoints like `base` and `md` are more informative than an array of values.